milestone.go

```diff
@@ -135,64 +135,66 @@ type Milestone struct {
 	Index uint32
 	// The time at which this milestone was issued.
 	Timestamp uint64
+	// The pointer to the previous milestone.
+	// Zeroed if there wasn't a previous milestone.
+	LastMilestone MilestoneID
 	// The parents where this milestone attaches to.
 	Parents MilestoneParentMessageIDs
-	// The inclusion merkle proof of included/newly confirmed transaction IDs.
-	InclusionMerkleProof MilestoneInclusionMerkleProof
-	// The next minimum PoW score to use after NextPoWScoreMilestoneIndex is hit.
-	NextPoWScore uint32
-	// The milestone index at which the PoW score changes to NextPoWScore.
-	NextPoWScoreMilestoneIndex uint32
-	// The public keys validating the signatures of the milestone.
-	PublicKeys []MilestonePublicKey
-	// The inner payload of the milestone. Can be nil or a Receipt.
-	Receipt serializer.Serializable
+	// The merkle root of all directly/indirectly referenced messages (their IDs) which
+	// were newly confirmed by this milestone.
```

Comment:
We call that "referenced" in the hornet codebase. :)

Reply:
You mean `// were newly confirmed by this milestone.` to `// were newly referenced by this milestone.`?